conda_forge_tick/migrators/arch.py

```diff
@@ -222,7 +222,7 @@ def __init__(
             self.graph.remove_nodes_from([n for n in self.graph if n not in packages])
 
         for name in self.excluded_dependencies:
-            self.graph.remove_nodes_from(nx.descendants(graph, name))
+            self.graph.remove_nodes_from(nx.descendants(self.graph, name))
```

Comment:
The problem here is that very few feedstocks have fortran in their host deps, and currently the stub doesn't have a run_exports like impact, so this may include feedstocks that require a fortran compiler.